src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Delete align-items: center for the product-card, set correct width for the card images and then you can delete width: 100% for all block elements in your card